src/scripts/utils/ParatiiLib.js

```diff
@@ -29,3 +29,28 @@ if (process.env.NODE_ENV === 'test') {
 }
 
 export const paratii: ParatiiLib = window.paratii
+
+export async function setupKeystore () {
+  let defaultPassword = 'password'
```

Comment:
I would put a lot of these strings in a new `constants/ParatiiConstants.js` file